test/test_mcs.jl

```diff
@@ -10,35 +10,44 @@ confidence = 2.576 # 99% CI by default; use 1.96 to apply a 95% CI, but expect m
 # Monte Carlo distribution information
 # Filled in from a run with regenerate = true
 information = Dict(
-    :td => Dict(:transform => x -> log(x), :mu => 19.268649852193303, :sigma => 1.0189429437682878),
-    :tpc => Dict(:transform => x -> log(-x), :mu => 16.723536381185546, :sigma => 0.9100375935397476),
-    :tac => Dict(:transform => x -> log(x), :mu => 17.35110886191308, :sigma => 0.3586397403122257),
-    :te => Dict(:transform => x -> log(x), :mu => 19.30193679432187, :sigma => 1.048364875171592),
-    :c_co2concentration => Dict(:transform => x -> x, :mu => 913073.0003103315, :sigma => 71674.18878428967),
-    :ft => Dict(:transform => x -> x, :mu => 8.637901336461365, :sigma => 0.4315112767385383),
-    :rt_g => Dict(:transform => x -> x, :mu => 5.888175260413073, :sigma => 1.7673873280904946),
-    :sealevel => Dict(:transform => x -> x, :mu => 1.6056649448621665, :sigma => 0.6124270241884567)
+    :td => Dict(:transform => x -> log(x), :mu => 20.814327104215817, :sigma => 0.7007588482770571),
+    :tpc => Dict(:transform => x -> x, :mu => 3.23416123428041e7, :sigma => 6.374797685068551e7),
+    :tac => Dict(:transform => x -> log(x), :mu => 14.944386813804241, :sigma => 0.46207147966982504),
+    :te => Dict(:transform => x -> log(x), :mu => 20.84214660784536, :sigma => 0.6995498063474193),
+    :c_co2concentration => Dict(:transform => x -> x, :mu => 651134.5999874412, :sigma => 46363.450549201516),
+    :ft => Dict(:transform => x -> x, :mu => 6.216834969325794, :sigma => 0.4011489912917528),
+    :rt_g => Dict(:transform => x -> x, :mu => 4.366704203451731, :sigma => 1.219601225681128),
+    :sealevel => Dict(:transform => x -> x, :mu => 3.026419413461323, :sigma => 1.1459316083150264)
```

Comment:
Do these numbers match the results prior to this change? Like, could we cherry-pick this out into a separate PR and it would pass?

Reply:
That was the spirit of it, I created this file using the `regenerate = true` option _before_ any of the other changes to get the tests passing and then added the other tests.  I think it would be possible though to pull this out into another PR that makes sure to pin `DataFrames` and `Mimi` to lower versions so they don't cause errors, check these tests, merge, and then add this new PR on top that frees those dependencies and adds the other changes.

I can do that today/tomorrow if you'd like no problem!

Reply:
The commented out part of the file at the bottom starting with "Used to produce testing files on May 21, 2021 with master" shows where the testing data came from!  I can definitely try to make a cherry picked PR though it can't be that hard :)

Reply:
see #77

Reply:
I think ideally we get #77 passing and merged first, then we merge master into this branch, and merge this one.  I'll keep tabs on that.

Reply:
Amazing. Thanks for doing that!